src/main/java/uk/gov/hmcts/reform/hois/controllers/HoisController.java

```diff
@@ -30,16 +37,22 @@
 @RestController
 public class HoisController {
 
+    @Autowired
+    private transient ObjectMapper objectMapper;
+
+    @org.springframework.beans.factory.annotation.Value("classpath:homeoffice-api-response-mock.json")
```

Comment:
this is production code, I mean, it's a test class but you are naming this resourceResFile with the mock word in it. It seems a bit weird to a mock on code that goes to production. Is this OK?

Reply:
As of now, we do not have the external API from HomeOffice. For now, we have decided to stub it.